### PR TITLE
Fix: Updated hcp binary extraction procedure and enahancements

### PR DESCRIPTION
**Pull request by veera-damisetti**
- Updated hcp binary extraction procedure
- Removes os_variant  variable in bastion creation, as it can fetch from installation media 
- Updated documentation

### CODE REVIEW

File: docs/set-variables-group-vars.md [hunk 1]
```diff
@@ -317,7 +317,6 @@
 **hcp.bastion_params.file_server.ip** | IPv4 address for the file server that will be used to pass config files and iso to KVM host LPAR(s) and bastion VM during their first boot. | 192.168.10.201
 **hcp.bastion_params.file_server.protocol** | Protocol used to serve the files, either 'ftp' or 'http' | http
 **hcp.bastion_params.file_server.iso_mount_dir** | Directory path relative to the HTTP/FTP accessible directory where RHEL ISO is mounted. For example, if the FTP root is at /home/user1 and the ISO is mounted at /home/user1/RHEL/8.7 then this variable would be RHEL/8.7 - no slash before or after. | RHEL/8.7
-**hcp.bastion_params.os_variant** | rhel os variant for creating bastion | 8.7
```
Comment (isumitsolanki) [inline docs/set-variables-group-vars.md:320]: Hi @veera-damisetti,  Why are you removing os_variant ? How are you handling version specific bastion creation. ?

Comment (veera-damisetti) [reply docs/set-variables-group-vars.md:320]: As i am giving this flag, --osinfo detect=on
it should fetch from the installation media

**Review by AmadeusPodvratnik — APPROVED** [review]
Hi @veera-damisetti , thank you for the PR. /lgtm

**Pull request merged**